traffic_ops/testing/compare/compare.go

```diff
@@ -239,46 +257,48 @@ func (to *Connect) get(route string) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("Accept", "application/json")
+
+	// Should wait for any retries to complete before sending a request
+	to.mutex.Lock()
```

Comment:
rather than unlocking before each return,  better to `defer to.mutex.Unlock()` here..